catalog/rest/catalog-rest-impl/src/main/java/org/codice/ddf/rest/impl/CatalogServiceImpl.java

```diff
@@ -548,6 +548,14 @@ private BinaryContent createMetacard(
 
     } catch (MetacardCreationException | CatalogTransformerException e) {
       throw new CatalogServiceException("Unable to create metacard");
+    } finally {
+      try {
+        if (stream != null) {
+          stream.close();
+        }
+      } catch (IOException e) {
+        LOGGER.error("Unexpected error closing stream", e);
```

Comment:
❓ Do we want these at error? 
https://codice.atlassian.net/wiki/spaces/DDF/pages/7045131/Log+Levels